synfig-studio/src/synfigapp/vectorizer/centerlineadjustments.cpp

```diff
@@ -5,17 +5,30 @@
 **	$Id$
 **
 **	\legal
-**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**	This file uses code from OpenToonz open-source animation software  (https://github.com/opentoonz/opentoonz/), which is developed from Toonz, a software originally created by Digital Video, S.p.A., Rome Italy Digital Video, S.p.A., Rome Italy.
 **
-**	This package is free software; you can redistribute it and/or
-**	modify it under the terms of the GNU General Public License as
-**	published by the Free Software Foundation; either version 2 of
-**	the License, or (at your option) any later version.
+**	Copyright (c) 2016 - 2019, DWANGO Co., Ltd.
+**  Copyright (c) 2016 Toshihiro Shimizu - https://github.com/meso
```

Comment:
Please make sure to use "tab" instead of spaces to make this line aligned with previous ones. ^__^